src/main/java/jeeves/parser/Parser.java

```diff
@@ -0,0 +1,221 @@
+package jeeves.parser;
+
+import jeeves.exception.MissingByException;
+import jeeves.exception.MissingDescriptionException;
+import jeeves.exception.MissingFromException;
+import jeeves.exception.MissingIdException;
+import jeeves.exception.MissingToException;
+import jeeves.exception.NotIntegerIdException;
+import jeeves.exception.OutOfBoundIdException;
+import jeeves.task.Task;
+
+import java.util.ArrayList;
+
+/**
+ * Parser processes user input and returns an arraylist of string tokens for other classes to use.
+ */
+public class Parser {
+
+    private static final int FINDCOMMAND_TODO_OFFSET = 5;
```

Comment:
FIND_COMMAND_TODO_OFFSET

Reply:
Will update in a future commit, thanks for the code review